test/Libraries/DynamoPythonTests/PythonEditTests.cs

```diff
@@ -612,7 +613,31 @@ public void CPythonClassCanBeReturnedAndSafelyDisposedInDownStreamNode()
             ViewModel.Model.ExecuteCommand(deleteCmd2);
 
             Assert.IsEmpty(DynamoCPythonHandle.HandleCountMap.Where(x => x.ToString().Contains("myClass")));
+        }
+
+        [Test]
+        public void VerifySysPathValueForCPythonEngine()
+        {
+            // open test graph
```

Comment:
Can you also add an assertion or another simple test to make sure the number of paths is 3 before making any changes to it? Given those paths are added by Python.NET by default, and we are assuming they are 3 on our end, this would be useful to catch changes on their end.

Reply:
Added it to the same test.